### PR TITLE
Rewards -> Supercharge experiment

### LINKED ISSUE #1686: "Rewards" to "Supercharge"
This experiment changes the text “Rewards” (with Celo coins icon) to “Supercharge” (with lightning bolt icon); and replaces the pill with a button.

ENG:
* [Experiment design](https://www.notion.so/valora-inc/Rewards-to-Supercharge-2ff750ad77884ffbb2837c61525e3477)
* Firebase RemoteConfig item that configures the four cohorts.
* The component that holds the icon and text is called `RewardsPill`

AC:
- [ ] Control: Current “Rewards” with coins pill
- [ ] Cohort 0: “Supercharge” with lightning bolt pill
- [ ] Cohort 1: “Rewards” with coins hamburger menu item
- [ ] Cohort 2: “Supercharge” with lightning bolt hamburger menu item

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Set up an experiment to test different buttons to open the rewards screen.

### Other changes

N/A

### Tested

Manually trying each possible value

Screen Shot 2022-01-17 at 20 24 14
Screen Shot 2022-01-17 at 20 24 32
Screen Shot 2022-01-17 at 20 23 58
Screen Shot 2022-01-17 at 20 23 44

### How others should test

Hard to test the different values, not sure how to test without access to the code or to setting different values in remote config :/

### Related issues

- Fixes #1686

### Backwards compatibility

N/A

### CODE REVIEW

**Review by silasbw — APPROVED** [review]
:artist:

File: packages/mobile/src/app/saga.ts [hunk 1]
```diff
@@ -171,6 +172,7 @@ export interface RemoteConfigValues {
   linkBankAccountEnabled: boolean
   sentryTracesSampleRate: number
   sentryEnabled: boolean
+  superchargeButton: SuperchargeButton
```
Comment (jeanregisser) [inline packages/mobile/src/app/saga.ts:175]: nit:
```suggestion
  superchargeButtonType: SuperchargeButtonType
```

File: packages/mobile/src/app/types.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export enum SuperchargeButton {
```
Comment (jeanregisser) [inline packages/mobile/src/app/types.ts:1]: nit:
```suggestion
export enum SuperchargeButtonType {
```

**Review by jeanregisser — APPROVED** [review]
Great! 👍

**Pull request merged**